requirements.txt

```diff
@@ -1 +1 @@
-pytest==7.2.1
\ No newline at end of file
+pytest==7.4.3
```

Comment:
Given the somewhat artificial nature (small code base and little communication) of this exercise and the small size of the change, keeping this change as an initial commit in the PR instead of a separate PR seems appropriate.